unittests/allplatformstests.py

```diff
@@ -1083,7 +1083,7 @@ def test_always_prefer_c_compiler_for_asm(self):
         for cmd in self.get_compdb():
             # Get compiler
             split = split_args(cmd['command'])
-            if split[0] == 'ccache':
+            if split[0] in ('ccache', 'sccache'):
```

Comment:
```suggestion
            if split[0].endswith('ccache'):
```

That also covers the case of it's a full path.

Reply:
Which would only happen if the user is somehow overriding the compiler(s), no? Is that considered a valid use case for the testsuite?

Reply:
Just to be clear, `sccache` gets automatic priority over `ccache`, so just having it installed triggers the failure.